db/migrate/20190612144842_add_user_attributes.rb

```diff
@@ -0,0 +1,8 @@
+class AddUserAttributes < ActiveRecord::Migration[5.2]
+  def change
+    add_column          :users, :gender, :integer 
+    change_column_null  :users, :gender, false   
```

Comment:
You can remove this and add to the previous line this: `null: false` so it should look like this
```add_column          :users, :gender, :integer, null: false```